src/qt/psbtoperationsdialog.cpp

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2011-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qt/psbtoperationsdialog.h>
+
+#include <core_io.h>
+#include <interfaces/node.h>
+#include <key_io.h>
+#include <node/psbt.h>
+#include <qt/bitcoinunits.h>
+#include <qt/forms/ui_psbtoperationsdialog.h>
+#include <qt/guiutil.h>
+#include <qt/optionsmodel.h>
+#include <util/strencodings.h>
+
+#include <iostream>
+
+size_t CountPSBTUnsignedInputs(const PartiallySignedTransaction &psbt) {
```

Comment:
Maybe move this to `psbt.h`?

Reply:
Can do, was dithering about what to do with it.